package.json

```diff
@@ -60,7 +60,7 @@
     "inferno-router": "^8.2.3",
     "inferno-server": "^9.0.0",
     "jwt-decode": "^4.0.0",
-    "lemmy-js-client": "0.20.0-alpha.17",
+    "lemmy-js-client": "1.0.0-alpha.0",
```

Comment:
Not sure that we should do this yet, and compeletely break main. Probably safer to do PRs to my 1.0 dev branch until we get everything compiling.

https://github.com/LemmyNet/lemmy-ui/pull/2985

Reply:
Agree, otherwise it will be impossible to work on lemmy-ui until that PR is merged.